doc/introduction/importing_circuits.rst

```diff
@@ -0,0 +1,267 @@
+0: ──H─╭●─╭||─┤  <Y>
+1: ────╰X─╰||─┤  Var[Z]
+
+OpenQASM
+~~~~~~~~
+
+An equivalent quantum circuit can be expressed in OpenQASM 2.0 as follows:
+
+.. code-block:: python
+
+    oq_circuit = (
+        """
+        OPENQASM 2.0;
+        include "qelib1.inc";
+        qreg q[2];
+        creg c[2];
+
+        h q[0];
+        cx q[0], q[1];
+
+        measure q -> c;
+        """
+    )
+
+We can import this circuit into PennyLane using the PennyLane-Qiskit plugin once more:
+
+.. code-block:: python
+
+    import pennylane as qml
+
+    pl_template_from_oq = qml.from_qasm(oq_circuit)
+
+    @qml.qnode(qml.device("default.qubit"))
+    def pl_circuit_from_oq():
+        pl_template_from_oq(wires=[0, 1])
+        return qml.expval(qml.Y(0)), qml.var(qml.Z(1))
+
+The result is as follows:
+
+>>> print(qml.draw(pl_circuit_from_oq)())
+0: ──H─╭●──┤↗├─┤  <Y>
+1: ────╰X──┤↗├─┤  Var[Z]
+
+Quil
+~~~~
+
+PennyLane also offers convenience functions for importing circuits from `pyQuil
+<https://pyquil-docs.rigetti.com/en/stable/index.html>`__ or Quil representations. Both of these
+require the `PennyLane-Forest <https://docs.pennylane.ai/projects/rigetti/en/stable/>`__ plugin,
+which can be installed using:
+
+.. code-block::
+
+    pip install pennylane-forest
+
+We begin with a familiar pyQuil `Program
+<https://pyquil-docs.rigetti.com/en/stable/apidocs/pyquil.quil.html#pyquil.quil.Program>`__:
+
+.. code-block:: python
+
+    import pyquil
+
+    pq_program = pyquil.Program()
+    pq_program += pyquil.gates.H(0)
+    pq_program += pyquil.gates.CNOT(0, 1)
+
+This ``Program`` can be converted into a PennyLane quantum function using the :func:`from_pyquil`
+function:
+
+.. code-block:: python
+
+    import pennylane as qml
+
+    pl_template_from_pq = qml.from_pyquil(pq_program)
+
+    @qml.qnode(qml.device("default.qubit"))
+    def pl_circuit_from_pq():
+        pl_template_from_pq(wires=[0, 1])
+        return qml.expval(qml.Y(0)), qml.var(qml.Z(1))
+
+The resulting PennyLane circuit is:
+
+>>> print(qml.draw(pl_circuit_from_pq)())
+0: ──H─╭●─┤  <Y>
+1: ────╰X─┤  Var[Z]
+
+.. note::
+
+    Quantum circuits expressed in Quil can be imported in a similar way using :func:`from_quil`.
+
+
+Importing Quantum Operators
+---------------------------
+
+Sometimes, it is preferable to import a single operation from a framework instead of an entire
+quantum circuit. This can save you some keystrokes and serve as a helpful crutch for understanding
+an individual component of a circuit.
+
+Presently, only Qiskit `SparsePauliOp
+<https://docs.quantum.ibm.com/api/qiskit/qiskit.quantum_info.SparsePauliOp>`__ operators can be
+imported into PennyLane. To see this in action, we first define a ``SparsePauliOp``:
+
+.. code-block:: python
+
+    from qiskit.quantum_info import SparsePauliOp
+
+    qk_op = SparsePauliOp(["II", "XY"])
+
+Then, we apply the :func:`from_qiskit_op` function to convert the ``SparsePauliOp`` into a PennyLane
```

Comment:
The `from_qiskit_op` link is not working

Reply:
Fixed! It should be ``` :func:`~pennylane.from_qiskit_op` ``` and similarly below. Thanks! 🎣